src/main/java/com/github/kongchen/swagger/docgen/reader/JaxrsReader.java

```diff
@@ -194,6 +198,24 @@ public String value(){
         return filteredMethods;
     }
 
+    private void readCommonParameters(Class<?> cls) {
+        Path path = AnnotationUtils.findAnnotation(cls, Path.class);
```

Comment:
> One possibility would be to annotate a Class with io.swagger.annotations.Api but without any javax.ws.rs.Path annotation, to indicate that this class is not specific to any path and only contains "common/global" parameters.

I believe that we should also look on methods for a Path annotation to determine if a given class is resource class.
Also, we should check if the method is annotated with any of the HTTP Method Verbs (*PUT*, *GET*, *POST*, etc..).
Compare this with the document *JSR-000370 JAX-RS 2.1 Final Release Specification for Evaluation*, section 3.1:

>Resource classes are POJOs that have at least one method annotated with@Path or a request method desig-
nator.



See method extractOperationMethod in JaxrsReader.